src/main/features/dashboard/views/index.njk

```diff
@@ -44,25 +44,13 @@
             </tr>
           {% endif %}
           {% for claim in claimsAsClaimant %}
-            {% if claim.eligibleForCCJ %}
-              {% set status = t('Request a County Court Judgment.') %}
-            {% elseif claim.countyCourtJudgmentRequestedAt %}
-              {% set status =
-              t('You requested a County Court Judgment on %s. We will contact you within 5 days.',
-              { postProcess: 'sprintf', sprintf: [ claim.countyCourtJudgmentRequestedAt | date ]})
-              %}
-            {% else %}
-              {% set status = t('Your claim is being processed. We will email you to explain what to do next.') %}
-            {% endif %}
-
-
             <tr class="claims default mobile-table">
               <td><a href="/dashboard/{{ claim.externalId }}/claimant">{{ claim.claimNumber }}</a></td>
               <td>{{ claim.claimData.defendant.name }}</td>
               <td>{{ claim.totalAmount | numeral }}</td>
-              <td class="mobile-hide">{{ status }}</td>
+              <td class="mobile-hide">{{ responseStatusForClaimant(claim) }}</td>
             </tr>
-            <tr><td colspan="3" class="mobile-table-status">{{ t('Status: ') + status }}</td></tr>
+            <tr><td colspan="3" class="mobile-table-status">{{ t('Status: ') + responseStatusForClaimant(claim) }}</td></tr>
```

Comment:
may be {{ t('Status: %s', { postProcess: 'sprintf', sprintf: [ responseStatusForClaimant(claim) ]})}}

Reply:
Not sure that would work, `responseStatusForClaimant ` does not return anything, just prints HTML markup. Worth trying, was curious myself if that would work.